lighthouse-cli/test/smokehouse/report-assert-test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {compareChromiumVersions} from './report-assert.js';
+
+/* eslint-env jest */
+
+describe('report-assert', () => {
+  it('compares chromium versions correctly', async () => {
+    expect(compareChromiumVersions([100, 0, 0, 0], [100, 0, 0, 0])).toBe(0);
```

Comment:
ideally if we're going to be exporting a function and testing it, it would be the whole kit and caboodle, like `expect(failsChromeVersionCheck(hostVersion, {_minChromiumVersion: '100'}).toBe(whatever)`

Reply:
done